examples/ATC/publisher.cpp

```diff
@@ -0,0 +1,149 @@
+/*                CycloneDDS isoC++ example 
+ *
+ * Copyright(c) 2006 to 2018 ADLINK Technology Limited and others
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Eclipse Distribution License v. 1.0 which is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ * SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
+ *
+ */
+
+#include <iostream>
+#include "dds/dds.hpp"
+#include "dds/core/ddscore.hpp"
+#include "ATCDataModel_DCPS.hpp"
```

Comment:
@RamziKaroui may you can update the source code too for the current master. There is no _DCPS anymore. 
And no need to include dds/core/ddscore, its already included in dds/dds.hpp